api/src/main/java/datawave/webservice/request/objects/ReferencedValue.java

```diff
@@ -82,7 +83,7 @@ public void setValue(String value) {
         if (isValidXML(value)) {
             this.value = value;
         } else {
-            this.value = new String(Base64.encodeBase64(value.getBytes(Charset.forName("UTF-8"))));
+            this.value = new String(Base64.encodeBase64(value.getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
```

Comment:
FWIW, I find it very useful to statically import the `UTF_8` field, so these things can be shorter. Guava also had a `UTF_8` instance, and it was really nice to transition from Guava to the Java built-in by only changing the static import. Also, the line is shorter when the import is static. I know sometimes they are discouraged, but this is one of those rare cases, I think, where it's a good idea to use static imports.

Reply:
Thanks for the feedback! Added.